geth/notification/notification.go

```diff
@@ -0,0 +1,45 @@
+package notification
+
+import (
+	"github.com/status-im/status-go/geth/common"
+	"github.com/status-im/status-go/geth/log"
+)
+
+// Manager of push notifications
+type Manager struct {
+	messaging common.Messaging
+}
+
+// New notifications manager
```

Comment:
`New returns new instance of notification manager`?

Reply:
Yep.The usual call looks like 'notification.New()'. I think it tells by itself.